### ISSUE TITLE
IPlong should be a Int64 instead on Int

### ISSUE THREAD

**Issue by yvesago** [ns3777k/go-shodan#27]
### Issue details and expected behavior

When we build a 32 bits app  the max size of int is 2147483647 
So address after 127.255.255.255 make an error.

A simple patch could be :
```
diff --git a/shodan/host.go b/shodan/host.go
index 8c4d023..5f0914b 100644
--- a/shodan/host.go
+++ b/shodan/host.go
@@ -64,7 +64,7 @@ type HostData struct {
        Hostnames    []string               `json:"hostnames"`
        Version      HostVersion            `json:"version"`
        Title        string                 `json:"title"`
-       IPLong       int                    `json:"ip"`
+       IPLong       int64                  `json:"ip"`
        IP           string                 `json:"ip_str"`
        OS           string                 `json:"os"`
        Organization string                 `json:"org"`
@@ -89,7 +89,7 @@ type HostData struct {
 type Host struct {
        OS              string      `json:"os"`
        Ports           []int       `json:"ports"`
-       IPLong          int         `json:"ip"`
+       IPLong          int64       `json:"ip"`
        IP              string      `json:"ip_str"`
        ISP             string      `json:"isp"`
        Hostnames       []string    `json:"hostnames"`

```

1. What `go` version are you using?

go version go1.9.2

2. Are you using the latest `go-shodan` version?
```go
import github.com/ns3777k/go-shodan/shodan
```

3. What OS are you using?

### Minimal reproducible demo

```bash
$ git clone https://github.com/yvesago/shodan-cli.git
$ GOARCH=386 go build

$ export SHODAN=xxxxxxYourAPIkeyxxxxxxxxxxxx

$ ./shodan-cli -i 192.169.138.132
Error: json: cannot unmarshal number 3232336516 into Go struct field Host.ip of type int

```

**Comment by ns3777k** [comment]
Hey @yvesago ! Thanks for creating an issue, I'll take a look soon.

**Comment by ns3777k** [comment]
I've applied your patch and released v2.0.2, thank you! @yvesago Can you verify everything's fine now?

**Comment by yvesago** [comment]
Perfect!
I made test with 
```go
import gopkg.in/ns3777k/go-shodan.v2/shodan
```
Everything is fine.

Thanks for the patch and your useful package
